<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuyDlg</class>
 <widget class="QDialog" name="BuyDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit { 
    border: 1px solid rgb(41, 57, 85);
    background: white;
    selection-background-color: darkgray;
    font-size: 16px ;
}

QLineEdit:hover {
	border: 2px solid blue;
}

QLineEdit[echoMode=&quot;2&quot;] {
      lineedit-password-character: 42;
}

QPushButton{
	border-radius:6px;
	background-color: #1390C5;
	min-width:80px;
}

QPushButton:pressed {
	background-color: #105D7E;
}

QPushButton:hover {
	background-color: #2BA0F3;
}

color: rgb(211, 211, 211);

</string>
  </property>
  <widget class="QLineEdit" name="paymentEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>290</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="useridEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>380</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PayBtn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>380</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>确认付款</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="orderidEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="timeEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>46</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>71</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font: 25 11pt &quot;微软雅黑 Light&quot;;</string>
      </property>
      <property name="text">
       <string>订单号:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font: 25 11pt &quot;微软雅黑 Light&quot;;</string>
      </property>
      <property name="text">
       <string>书名:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">font: 25 11pt &quot;微软雅黑 Light&quot;;</string>
      </property>
      <property name="text">
       <string>用户名:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">font: 25 11pt &quot;微软雅黑 Light&quot;;</string>
      </property>
      <property name="text">
       <string>时间:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">font: 25 11pt &quot;微软雅黑 Light&quot;;</string>
      </property>
      <property name="text">
       <string>数量:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">font: 25 11pt &quot;微软雅黑 Light&quot;;</string>
      </property>
      <property name="text">
       <string>需付款:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>BuyDlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>450</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
